README.md

```diff
@@ -90,6 +92,7 @@ module "gke" {
 }
 ```
 
+<!-- do not understand what this is about -->
```

Comment:
These are just the steps to run Terraform. In theory you could copy & paste the above config into a Terraform file then follow these steps to deploy the GKE cluster.

Reply:
I will add what you wrote and it will make more sense to me.